AncientReligions/history/characters/irish.txt

```diff
@@ -23509,7 +23509,10 @@
 	culture="irish"
 	father = 7258
 	550.1.1={
-		birth="550.1.1" 
+		birth="550.1.1"
+		effect = {
+			set_global_flag = ancrel
```

Comment:
It's something zijistark showed me. The paganify decision will only be available so long as the ancrel flag has existed for less than a day. Even though the flag was set hundreds of years before the first start date, it will be treated as having been set on the first day of the game.

This is how the EMF and PB customization decisions are only available on the first day of a new game.